lib/json_tagged_logger/formatter.rb

```diff
@@ -22,7 +22,7 @@ def call(severity, _time, _progname, message)
         log[:tags] = text_tags.to_a
       end
 
-      bare_message = message_without_tags(message)
```

Comment:
@santry I think this is the only thing missing, but it is actually for compatibility with Rails 8, where sending `nil` no longer works, it expects a string.